scripts/torch/param_dictionary.txt

```diff
@@ -0,0 +1,212 @@
+feature.stage1.0.conv.weight, module.feature.conv0.conv.weight
```

Comment:
This whole file is a hack to allow loading the existing checkpoint from Pytorch. The names of the parameters do not match up exactly. Ideally there should be a checkpoint created with the correct names and then this "translation" can go away.